doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1409,9 +1409,8 @@ using the keyword :cmd:`Qed`.
 Attributes
 -----------
 
-Any vernacular command can be decorated with a list of attributes, enclosed
-between ``#[`` (hash and opening square bracket) and ``]`` (closing square bracket)
-and separated by commas ``,``.
+Any vernacular command can be decorated with a list of attributes, each enclosed
+between ``#[`` (hash and opening square bracket) and ``]`` (closing square bracket).
```

Comment:
Should you add "or separated by commas inside a single `#[` `]` block for the sake of completeness?